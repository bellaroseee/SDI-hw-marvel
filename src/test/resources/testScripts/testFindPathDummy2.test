# testing findPath in dummy2 graph

LoadGraph dummy2 dummy2.tsv

FindPath dummy2 Billings,_Ira Blade_Summer

FindPath dummy2 Bennet,_James Black_Manta

FindPath dummy2 Bennet,_James Blubber

FindPath dummy2 3G4 Bennet,_James