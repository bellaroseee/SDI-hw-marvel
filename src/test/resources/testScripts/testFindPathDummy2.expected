# testing findPath in dummy2 graph

loaded graph dummy2

path from Billings, Ira to Blade Summer:
Billings, Ira to Blade Summer via BB

path from Bennet, James to Black Manta:
Bennet, James to Bennie via BB
Bennie to Black Manta via A35

path from Bennet, James to Blubber:
Bennet, James to Bennie via BB
Bennie to Blubber via A35

path from 3G4 to Bennet, James:
no path found